docs/tutoriales/guia-markdown.md

```diff
@@ -13,24 +13,23 @@ Para mayor información puede dar click en el siguiente link [¿Qué es Markdown
 
 ## Sintaxis Básica
 
-| Sintaxis     | Detalle |
-| :----: | :----: | 
-| # H1    | Título   | 
-| ## H2    | Título  | 
-| ### H3  | Título | 
-| **** texto **** | Tipografía Negrita|
-| * * texto * * |Tipografía itálica|
-| texto + doble espacio| Salto de línea|
+| Componente | Sintaxis|
+| ----------- | ----------- |
+| [Encabezado](https://www.markdownguide.org/basic-syntax/#headings) |  # H1   <br /> ## H2 <br />  ### H3  |
+| [Tipografía Negrita](https://www.markdownguide.org/basic-syntax/#bold) |  `**Texto**` or `__texto__` |
+| [Tipografía Itálica](https://www.markdownguide.org/basic-syntax/#italic) |   `*texto*` or  `_texto_`|
+| [Citas en Bloque](https://www.markdownguide.org/basic-syntax/#blockquotes-1)    | > Citas en bloque  |
+| [Lista Numérica](#ordered-list) | 1. Primer Item <br /> 2. Segundo Item <br /> 3. Tercer Item  |
+| [Lista no Ordenada](#unordered-list) | - Primer Item <br /> - Segundo Item <br /> - Tercer Item |
+| [Linea Horizontal](#horizontal-rule) | --- |
+| [Enlaces](https://www.markdownguide.org/basic-syntax/#links) | `[título](https://www.example.com)` |  
+| [Imagen](https://www.markdownguide.org/basic-syntax/#images-1) | `![texto](image.jpg)` |
+|[Salto de línea](#line-break) | texto + doble espacio |
```

Comment:
I think, to have the same uppercase first letter format, in the phrase _Salto de línea_, the word could be **L**ínea

Reply:
Thanks!

Reply:
You welcome 🥇